### PR TITLE
Appointments show

### PR DESCRIPTION
**Pull request by jess-nash**

### CODE REVIEW

File: app/views/appointments/index.html.erb [hunk 1]
```diff
@@ -1 +1,4 @@
 <h1>appointments</h1>
+<%= @pet.appointments.each do |f| %>
```
Comment (noliver-ykc) [inline app/views/appointments/index.html.erb]: what is this?
Doesnt pushing this overwrite all my code?

**Review by noliver-ykc — CHANGES_REQUESTED** [review]
i think you need to delete index from your commit

File: app/views/appointments/index.html.erb (hunk 1 above)
Comment (noliver-ykc) [reply app/views/appointments/index.html.erb]: the github is displaying very strangely... if this problem is completly fixed then ill push it... but i cant tell lol

Comment (jess-nash) [reply app/views/appointments/index.html.erb]: That's because this was my first push before I pulled your index.

Comment (jess-nash) [reply app/views/appointments/index.html.erb]: @noliver-ykc please review the latest commit on this pr

Comment (jess-nash) [reply app/views/appointments/index.html.erb]: It shouldn't be deleted. That my code from before I pulled yours, also it's the index is on my local host so there's no way it was deleted.

**Review by k-ashmore — APPROVED** [review]
LGTM

**Pull request merged**